drools-wb-jcr2vfs-migration/drools-wb-jcr2vfs-import/pom.xml

```diff
@@ -180,6 +180,11 @@
       <artifactId>zip4j</artifactId>
       <scope>test</scope>
     </dependency>
+    <!-- Investigate why is gwt-user needed and ideally remove it, see https://issues.jboss.org/browse/DROOLS-1391 -->
```

Comment:
What's weird is that this is only needed when using JPA 2.1 + Hibernate 5.x. With the old JPA 2.0 + Hibernate 4.x it was working alright.